From 5d0828f9ffbcfbe92f8300e6fdd8e8bd737bdc50 Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Fri, 1 Jun 2018 09:41:02 +0800
Subject: [PATCH 4/4] arm64:dts:xilinx: add has-io-mode to workaround stack
 overflow

This patch is to workaround the stack overflow from driver level by using
IO mode instead of DMA.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
index 7c41c9b..42e4e62 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
@@ -846,6 +846,7 @@ drivers/hwmon/pmbus/Makefile:11:obj-$(CONFIG_SENSORS_MAX20751)  += max20751.o
 &qspi {
 	status = "okay";
 	is-dual = <1>;
+	has-io-mode;
 	flash@0 {
 		compatible = "m25p80"; /* 32MB */
 		#address-cells = <1>;
-- 
1.7.9.5

